test/README.md

```diff
@@ -12,7 +12,8 @@ To run only the integration tests, run from the test directory:
 
 Name                            | Default                   | Description
 :--                             | :---                      | :----------
-LXD\_BACKEND                    | dir                       | What backend to test against (btrfs, dir, lvm, zfs, or random)
+LXD\_BACKEND                    | dir                       | What backend to test against (btrfs, ceph, dir, lvm, zfs, or random)
+LXD\_CEPH\_CLUSTER              | ceph                      | The name of the ceph cluster to create osd pools in. Only used if "\${LXD_BACKEND}" = "ceph".
```

Comment:
Note, even if you set `LXD_BACKEND` to something else then `ceph` but set `LXD_CEPH_CLUSTER` nothing will happen wrt to `ceph`. All other backend test will just run fine as they would with `LXD_CEPH_CLUSTER` unset.